app/controllers/users_controller.rb

```diff
@@ -29,7 +30,8 @@ def update
     @user.update(user_params)
     if @user.save
       flash[:notice] = "Thine information has been updated!"
-      redirect_to dashboard_path
+      redirect_to dashboard_path if current_user.default?
```

Comment:
Matt... I appreciate the hot Latin flair that you bring to our team. However, I just don't see a need for that breakout. I had a private meeting with Katie and Ellen, behind your back, and we agree this was just fine before you suggested modifications.
Thanks... but no thanks.

*shrug*